layouts/partials/languageselector.html

```diff
@@ -0,0 +1,15 @@
+{{- if .IsTranslated -}}
+<div class="container md:flex justify-end md:gap-6 bg-gray-100 py-2">
+    <div class="inline-flex justify items-center gap-2">
+    {{- $currentLanguage := .Language -}}
+    {{- range $index, $page := sort .AllTranslations.Reverse -}}
+        {{ if ne $index 0 }}
+        <span class="border-l-2 h-3/4"></span>
+        {{- end -}}
+        {{- $lang := $page.Language -}}
+        {{$active := eq $currentLanguage $lang }}
+        <a href="{{ $page.RelPermalink }}" title="{{ $lang.LanguageName }}" class="{{ if $active }}text-blue-800 font-bold{{ else }}text-slate-300{{ end }}">{{ $lang.LanguageName }}</a>
```

Comment:
could it be a pull-down menu style? just wonder while I am dreaming we have much more supported languages :-)

Reply:
I think given we're only going to have 3 to 4 for now we should keep it like this. I'm hoping to work on using flags or something shorter. I think this one makes a lot more visible the languages the user will be able to call.